working-with-openai/open-ai-api-in-projects.md

```diff
@@ -339,43 +344,32 @@ def get_greetings(char_id):
             "greeting" : greeting.greeting_text
         })
     
-    return jsonify(response)
+    return response
 ```
 </details>
 </br>
 
 ## Summary
-In this lesson, we took a partially built API and expanded it to include AI generated content. This is just the tip of the iceberg when it comes to integrating the OpenAI API into your projects, so we encourage you as always to follow your curiosity and keep looking for new and exciting ways to bring AI into your projects!
+In this lesson, we took a partially built API and expanded it to include AI generated content. This is just the tip of the iceberg when it comes to integrating the AI APIs into your code, so we encourage you as always to follow your curiosity and keep looking for new and exciting ways to bring AI into your projects!
 
```

Comment:
Not sure if this is necessary, but would it be helpful to include a callout along the lines below?

```suggestion
### !callout-info

## Other AI APIs and Python
As mentioned before, Gemini is just one of many AI APIs you can integrate into your Python projects. We encourage you to explore others as your time and resources allow. While they won't look exactly the same, the steps for integration should be similar. There will likely be a Python package you can install with functions that will allow you to designate a model, attach your API key, make a request and more! Make sure to follow the documentation for whichever API you end up using and try things out!

### !end-callout  
```

Reply:
I think this is great, thank you for adding it!